addons/arsenal/defines.hpp

```diff
@@ -7,6 +7,14 @@
 #define WIDTH_GAP (WIDTH_TOTAL / 100)
 #define WIDTH_SINGLE ((WIDTH_TOTAL - 6 * WIDTH_GAP) / 5)
 
+// IDDs
+#define IDD_MISSION 46
+#define IDD_RSCDISPLAYCURATOR 312
+#define IDD_DISPLAY3DEN	313
```

Comment:
```suggestion
#define IDD_DISPLAY3DEN 313
```
> I can't find the tab that it's complaining about

it's right here :)

Reply:
I'm disappointed in myself that I hadn't found that - I was sure that I had tried to replace that with a whitespace.
Thanks!